src/main/java/co/kr/muldum/domain/dto/HistoryDetailResponseDto.java

```diff
@@ -0,0 +1,76 @@
+package co.kr.muldum.domain.dto;
+
+import co.kr.muldum.domain.history.model.History;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class HistoryDetailResponseDto {
+
+    private Long id;
+    private String name;
+    private Integer generation;
+    private String logoUrl;
+    private String description;
+    private String slogan;
+    private Detail detail;
+
+    @Getter
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class Detail {
+        private String background;
+        private String features;
+        private String research;
+        private List<Contributor> contributors;
+    }
+
+    @Getter
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class Contributor {
+        private String name;
+        private String githubUrl;
+    }
+
+    public static HistoryDetailResponseDto fromEntity(History history) {
+        if (history == null) return null;
+
+        List<Contributor> contributors = (history.getContributors() == null)
+                ? Collections.emptyList()
+                : history.getContributors().stream()
+                    .map(c -> Contributor.builder()
+                            .name(c.getName())
+                            .githubUrl(c.getGithubUrl())
+                            .build())
+                    .collect(Collectors.toList());
```

Comment:
```suggestion
                    .toList());
                    
```
로 변환하시는게 좋을 것 같고 사실 collector가 쓰이는 상황은 별로 많지 않습니다. 다음에 기회가 되시면 람다식, 스트림API, 함수형 프로그래밍에 대해 공부해보시는 것은 어떨까요?